### PR TITLE
reformat some headers

### PR DESCRIPTION
**Pull request by pjfanning**
I noticed a few license headers that were missing the `#` at the start of one line.

### CODE REVIEW

File: remote/src/main/mima-filters/1.1.x.backwards.excludes/netty4.backwards.excludes [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
 # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
 
-#migrate the classic transport to Netty4
+# migrate the classic transport to Netty4
 ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.pekko.remote.transport.netty.NettyFutureBridge.apply")
 ProblemFilters.exclude[MissingClassProblem]("org.apache.pekko.remote.transport.netty.ChannelLocalActor")
 ProblemFilters.exclude[MissingClassProblem]("org.apache.pekko.remote.transport.netty.ChannelLocalActor$")
```
Comment (mdedetrich) [inline remote/src/main/mima-filters/1.1.x.backwards.excludes/netty4.backwards.excludes:21]: While we are here can we add a new line?

**Review by mdedetrich — APPROVED** [review]
lgtm

File: remote/src/main/mima-filters/1.1.x.backwards.excludes/netty4.backwards.excludes (hunk 1 above)
Comment (pjfanning) [reply remote/src/main/mima-filters/1.1.x.backwards.excludes/netty4.backwards.excludes]: done

**Review by He-Pin — APPROVED** [review]
Lgtm

**Comment by mdedetrich** [comment]
Also remember to backport change to `1.0.x`

**Pull request merged**